docs-2.0/3.ngql-guide/11.edge-statements/4.delete-edge.md

```diff
@@ -0,0 +1,24 @@
+# DELETE EDGE
+
+```ngql
+DELETE EDGE <edge_type> <vid> -> <vid>[@<rank>]
```

Comment:
if more than one edges can be deleted, should we update this syntax with "DELETE EDGE <edge_type> <src_vid> -> <dst_vid> [, <src_vid> -> <dst_vid> ...]"? And, can "rank" be deleted?